src/Components/PhotoUpload/Components/PhotoThumbnail.tsx

```diff
@@ -87,6 +87,9 @@ export const PhotoThumbnail: React.FC<
     }
   }
 
+  // Only show error messages for photos that have been uploaded by the user
+  const showErrorMessage = photo.errorMessage && !photo.externalUrl
```

Comment:
I've decided to show regular messages here because the component is used in multiple places. Error messages with an external URL (automatically added) will be displayed in another place making sure only the last error message is shown.

Reply:
sounds good!